modules/network-observability-resource-recommendations.adoc

```diff
@@ -11,6 +11,10 @@ The following settings can help you manage resources and performance from the ou
 
 eBPF Sampling:: You can set the Sampling specification, `spec.agent.ebpf.sampling`, to manage resources. Smaller sampling values might consume a large amount of computational, memory and storage resources. You can mitigate this by specifying a sampling ratio value. A value of `100` means 1 flow every 100 is sampled. A value of `0` or `1` means all flows are captured. Smaller values result in an increase in returned flows and the accuracy of derived metrics. By default, eBPF sampling is set to a value of 50, so 1 flow every 50 is sampled. Note that more sampled flows also means more storage needed. Consider starting with the default values and refine empirically, in order to determine which setting your cluster can manage.
 
+eBPF features:: The more features that are enabled, the more impact it has on performances. See "Observing the network traffic" for a complete list of these features.
+
+Without Loki:: You can reduce the amount of resources that Network Observability requires by not using Loki and instead relying on Prometheus. For example, when Network Observability is configured without Loki with a `sampling` value of `50`, the total savings of memory usage are in the 20-65% range and CPU utilization is lower by 10-30%.
+
```

Comment:
This needs a link to point them to the limitations of not having Loki.

Reply:
In a comment to Mehul, I mention that we have hyperlink limitations within module files (as in we can't do it). So I will create an Additional Resources section and put the link to this there, and will add a quoted mention of "Without Loki" section in the text of the module.